_episodes/02-git.md

```diff
@@ -155,7 +185,8 @@ in the repository directory.
 
 #### git add, git status, git commit
 
-Making a commit is like making a checkpoint for a particular version of your code. You can easily return to, or revert to that checkpoint.
+Making a commit is like making a checkpoint for a particular version of your code.
+You can easily return to, or revert to that checkpoint.
 
 To create the checkpoint, we first have to make changes to our project. We might modify *many* files at a time in a repository. Thus, the first step in creating a checkpoint (or commit) is to tell `git` which files we want to include in the checkpoint. We do this with a command called `git add`. This adds files to what is called the *staging area*.
```

Comment:
Can you separate this line into multiple lines?

Reply:
I was trying to be minimally invasive, but it doesn't take that long to apply line breaks to the whole file. I'll go back and add them.